lib/miq_expression.rb

```diff
@@ -805,8 +805,9 @@ def self.quote_human(val, typ)
     case typ&.to_sym
     when :integer, :decimal, :fixnum, :float
       return val.to_i unless val.to_s.number_with_method? || typ == :float
+      return val if val.kind_of?(Numeric)
```

Comment:
Actually, we should move this up one line, right?  Why bother checking the previous line?

Reply:
yes

Reply:
good 👀

Reply:
done